tools/arch-visualizer/viewer/src/components/TreeNavigator.tsx

```
@@ -0,0 +1,111 @@
+        onClick={() => selectComponent(component.id)}
+        onDoubleClick={() => hasChildren && drillInto(component)}
+      >
+        {/* Expand/collapse */}
+        {hasChildren ? (
+          <span
+            className={`w-4 h-4 flex items-center justify-center text-[10px] shrink-0 ${darkMode ? "text-zinc-500" : "text-zinc-400"}`}
+            onClick={(e) => {
+              e.stopPropagation();
+              setExpanded(!expanded);
+            }}
+          >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)